src/ui.ts

```diff
@@ -118,7 +118,7 @@ export type PrivacyRegimeToInitialViewStateInput = t.TypeOf<
 >;
```

Comment:
I'm not sure what `PrivacyRegimeToInitialViewStateInput` is used for and if it similarly needs to be changed.

Reply:
@MadDataScience yes

Reply:
If you look for its usage in the monorepo you'll be able to get some idea of all the places where it's used. But the same thing applies--we should probably just relax the validation on the JSONB given that we're adding this new view state + because experiences are arbitrary anyway

Reply:
I don't know if `PrivacyRegimeToInitialViewStateInput` is still relevant. I decided, rather than replace, I'd create a new type, `ExperienceToInitialViewState`, and deprecate the old one. What do you think of that?